### DISCUSSION TITLE
Resisting Flechettes

### DISCUSSION THREAD

**Discussion by dlittle2257** [craigfowler/ShadowRunCampaign#3] [Q&A] [answered]
While I'm sure it doesn't affect the  Armour Penetration rule modification the description example has Flechette Rounds being resisted by ballistic armor. Page 323/324 have Flechette Rounds being resisted by Impact armor are you changing this?

**Comment by craigfowler** [comment, marked as answer]
Nope, this was a mistake. I have corrected the examples.
